### DISCUSSION TITLE
Task Manager with old layout shown when executing from code

### DISCUSSION THREAD

**Discussion by NielsKoomans97** [dotnet/csharplang#6645] [Show and tell]
I recently installed the newest Windows 11 update with the new Task Manager layout. Yesterday I encountered a weird bug where Task Manager has the older layout when launched from code.

![oldtaskmanager](https://user-images.githubusercontent.com/44090733/199454076-28680570-f8a1-4cc3-bec6-fb8cbf70ec4a.gif)

Is there a reason for this or am I just plain stupid?

**Comment by Richiban** [comment]
This is a repository for the design of the C# language itself; there's a C# discord at https://discord.com/channels/143867839282020352/1007030034902548540 which is a better place to ask for help with something.

**Reply by NielsKoomans97** [reply]
thank you very much!

can you perhaps provide me with an invite or an actual name of the discord, because i can't really find any info related to the link you gave me

**Reply by jnm2** [reply]
@NielsKoomans97 Sure thing! Check out https://github.com/dotnet/csharplang/blob/main/Communities.md which has invite links.

**Comment by GeeLaw** [comment]
Psychic debugging says your app is 32-bit hence launches the 32-bit task manager, which appears to use the old UI in 64-bit systems.

If you try (don't) removing all appx packages from the account, the task manager will also use the old UI, so it's likely that the appx package responsible for the new UI is 64-bit only.
